kie-dmn-feel/pom.xml

```diff
@@ -109,15 +105,6 @@
         </executions>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.5.1</version>
```

Comment:
The source/target levels are being explicitly configured here to 1.8. The config is just redundant as the same already comes from parent.